src/ngraph/runtime/ie/unit_test.manifest

```diff
@@ -0,0 +1,1026 @@
+
+# Cannot cast ngraph node GRUCell to CNNLayer!
+gru_cell_bias_clip
+gru_cell_linear_before_reset
+gru_cell_activation_function
+
+# Cannot cast ngraph node CrossEntropy to CNNLayer!
+cross_entropy_with_soft_labels
+cross_entropy_with_one_hot
+
+# Cannot cast ngraph node EmbeddingLookup to CNNLayer!
+embedding_lookup_4x5_reverse
+embedding_lookup_10x1_arbitrary
+embedding_lookup_10x1_arbitrary_index_type_int
+embedding_lookup_10x1_arbitrary_index_type_int64
+
+# Cannot cast ngraph node Dot to CNNLayer!
+dot_4d_5d_multi_axis
+dot_4d_5d_multi_axis_more
+dot1d
+dot2d
+dot2d_non_square
+dot3d_3d
+dot3d_2d
+dot_scalar_tensor_arg0
+dot_scalar_tensor_arg1
+dot_scalar_scalar
+dot_matrix_vector_4_3
+dot_matrix_vector
+dot_matrix_vector_int64
+dot_3d_multi_axis
+dot_3d_one_axis_arbitrary
+
+# Cannot cast ngraph node CumSum to CNNLayer!
+cum_sum_default
+cum_sum_2dim
+cum_sum_3d
+cum_sum_2dim_allmodes
+
+# Cannot create MKLDNNMemoryDesc from TensorDesc. Unsupported precision!
+convert_uint16_float32
+
+# Unsupported primitive of type: Ceiling
+ceiling
+
+# Cannot cast ngraph node BatchMatMul to CNNLayer!
+batch_mat_mul_forward
+
+# Incorrect dimensions for broadcasting for Add
+auto_bcast_binary_elementwise_pdpd_dynamic
+
+# Unsupported primitive of type: Atan2
+atan2
+
+# Unsupported primitive of type: ArgMin
+argmin_trivial
+argmin_2D_i32
+argmin_3D_i32
+argmin_3D_i64
+argmin_4D_i64
+argmin_4D_axis_3_i64
+argmin_4D_axis_3
+
+# Unsupported primitive of type: ArgMin
+argmin_trivial_in_i32
+
+# Error of validate layer: ArgMax with type: ArgMax. No such parameter name 'top_k' for layer ArgMax
+argmax_trivial
+argmax_2D_i32
+argmax_3D_i32
+argmax_3D_i64
+argmax_4D_i64
+argmax_3D_axis_0
+argmax_3D_axis_1
+argmax_3D_axis_2
+argmax_4D_axis_3
+argmax_4D_axis_3_i64_in_i32
+
+# Cannot find blob with name: Broadcast_1431690
+broadcast_algo_vector_middle
+broadcast_algo_vector_forward_3
+broadcast_algo_vector_forward_4
+broadcast_algo_vector_backward_3
+broadcast_algo_vector_backward_4
+broadcast_algo_matrix_backward_4
+broadcast_algo_matrix_stride_1
+broadcast_algo_matrix_stride_2
+broadcast_algo_matrix_stride_3
+
+# Cannot cast ngraph node ReplaceSlice to CNNLayer!
+# Incorrect precision u32!
+# Parameter has zero dimension that is not allowable
+dyn_replace_slice
+
+# Cannot cast ngraph node Reshape to CNNLayer!
+# Parameter has zero dimension that is not allowable
+flatten
+
+# Cannot cast ngraph node BatchMatMulTranspose_783933 to CNNLayer!
+batch_mat_mul_transpose
+batch_mat_mul_transpose_with_transpose
+
+# Failing from new reason after unblocking more Blob types
+gather_2d_negative_and_positive_indices_no_axis_2d_input
+gather_no_axis_int8
+gather_no_axis_uint8
+gather_no_axis_uint32
+squared_difference_broadcast
+split_3_equal_parts
+split_var_len_parts
+floor_int32
+strided_slice_0
+strided_slice_1
+strided_slice_stride_optional
+divide_int32
+divide_cpp_rounding_int32
+divide_python_rounding_int32
+convert_int32_bool
+lesseq_int32
```

Comment:
Who will track this file later and keep it up to date?

Reply:
Anyone working on ngraph code who introduces a new test, breaking change or anything else that makes a unit test on IE backend should add this test name here. There are similar manifests for other backends and for various reasons we have to exclude certain tests in them.

Reply:
I am not 100% sure that all of developers/testers know why some test was added here and when it is possible to remove it. This is so long file and not the easiest way to filter test on backend. However, as far as I know this is the only way.